model/src/main/java/com/basistech/rosette/dm/TranslatedData.java

```diff
@@ -30,26 +30,59 @@
 @EqualsAndHashCode(callSuper = true)
 public class TranslatedData extends BaseAttribute implements Serializable {
     private static final long serialVersionUID = 250L;
+    private final TextDomain sourceDomain;
+
+    /**
+     * @deprecated Replaced by targetDomain
+     */
+    @Deprecated(forRemoval = true)
     private final TextDomain domain;
+    private final TextDomain targetDomain;
     private final String translation;
     private final Double confidence;
 
-    protected TranslatedData(TextDomain domain, String translation, Double confidence, Map<String, Object> extendedProperties) {
```

Comment:
Don’t remove this constructor.

Reply:
It won't show as outdated but I added this back in line 44